### ISSUE TITLE
Just some tests

### ISSUE THREAD

**Issue by NarHakobyan** [joschan21/profanity.dev#27]
<img width="1316" alt="image" src="https://github.com/joschan21/profanity.dev/assets/1781506/b9c46622-51d5-4ce8-a4d5-1d25107c54b5">

**Comment by GwartneyDev** [comment]
@NarHakobyan Just curious should it catch the fk in the sentence? In this case or should it stay.

**Comment by NarHakobyan** [comment]
Yes, that's the way of saying in a way that social platforms can't catch it
